pkgs/by-name/mn/mni_autoreg/package.nix

```diff
@@ -30,6 +30,11 @@ stdenv.mkDerivation {
     MNI-Perllib
   ];
 
+  NIX_CFLAGS_COMPILE = toString [
```

Comment:
nit: `env.NIX_CFLAGS_COMPILE` is preferred.

Reply:
Good catch, fixed.